project_forecast_line/models/forecast_line.py

```diff
@@ -90,10 +90,12 @@ def _compute_employee_forecast_line_id(self):
         employees = self.mapped("employee_id")
         date_froms = self.mapped("date_from")
         date_tos = self.mapped("date_to")
+        forecast_roles = self.mapped("forecast_role_id")
```

Comment:
```suggestion
        forecast_roles = self.mapped("forecast_role_id") | employees.mapped("main_role_id")
```
otherwise the lookup on line 121 may give False